Dockerfile.ocp

```diff
@@ -1,6 +1,6 @@
-FROM rhel8:latest
+FROM registry.access.redhat.com/ubi8/ubi:latest
```

Comment:
let's keep this consistent with the other images and just put 'FROM ubi8' here

Reply:
There’s a bunch that do this too, like baremetal-runtimecfg. I could be wrong but I thought using it in this way makes sure we always pull the latest image instead of some local copy that could be quite old